### PR TITLE
BC-7618 - Missing breadcrumbs

### PR DESCRIPTION
**Pull request by muratmerdoglu-dp**
# Description

## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-7618

## Approval for review

- [ ] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: views/lib/loggedin.hbs [hunk 1]
```diff
@@ -255,6 +255,17 @@
                     <div class="container-fluid">
                         <div class="row" id="titlebar">
                             <div class="col-sm-9">
+																<div style="margin-bottom: 0.4rem; font-size: 14px;">
```
Comment (virgilchiriac) [inline views/lib/loggedin.hbs:258]: fix formating

File: views/lib/loggedin.hbs [hunk 2]
```diff
@@ -121,7 +121,7 @@
                         <a class="sc-branding" href="/" title="{{$t "global.link.landing"}}" data-testid="{{$t "global.link.landing"}}">
                             <div class="cloud-logo"></div>
                         </a>
-                        {{#if breadcrumb}}
+                        {{!-- {{#if breadcrumb}}
```
Comment (virgilchiriac) [inline views/lib/loggedin.hbs:124]: should be removed instead of commented

File: views/lib/loggedin.hbs (hunk 1 above)
Comment (muratmerdoglu-dp) [reply views/lib/loggedin.hbs]: fixed

File: views/lib/loggedin.hbs (hunk 2 above)
Comment (muratmerdoglu-dp) [reply views/lib/loggedin.hbs]: removed

File: views/lib/loggedin.hbs [hunk 3]
```diff
@@ -252,41 +240,52 @@
...
-                    </div>
+									<div class="container-fluid">
+										<div class="row" id="titlebar">
+											<div class="col-sm-9">
+												{{#if this.breadcrumbs}}
+													<div style="margin-bottom: 0.4rem; font-size: 14px;">
+														<span>
+															{{#each this.breadcrumbs}}
+																{{#if this.title}}
+																	<a class="{{#if @last}}active{{/if}}" href="{{this.url}}">{{{stripOnlyScript this.title}}}  </a> /
```
Comment (virgilchiriac) [inline views/lib/loggedin.hbs:251]: remove the space before closing </a>
you could add remove the space and `&nbsp; / &nbsp;` but that is also not good.... best is to use css

**Review by virgilchiriac — APPROVED** [review]

**Review by odalys-dataport — APPROVED** [review]

**Pull request merged**